js/apps/admin-ui/cypress/e2e/masthead_test.spec.ts

```diff
@@ -64,6 +66,48 @@ describe("Masthead tests", () => {
     });
   });
 
+  describe("Login works for unprivileged users", () => {
+    const realmName = `test-realm-${uuid()}`;
+    const username = `test-user-${uuid()}`;
+
+    before(async () => {
+      await adminClient.createRealm(realmName, { enabled: true });
+
+      console.info("Creating user" + username);
+      await adminClient.inRealm(realmName, () =>
+        adminClient.createUser({
```

Comment:
Should the promise from `createUser` not be returned? This way `inRealm` can await it.
```suggestion
        return adminClient.createUser({
```

Reply:
Since this is not a block but a lambda, it is done implicitly.

Reply:
It is actually not if you put brackets around the body, see the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body).

Reply:
There are no brackets. Thus it is expression, not a statement.

Reply:
Ah sorry, I misread.